index.js

```diff
@@ -4,7 +4,14 @@ var semver = require('semver-regex');
 var ZSchema = require('z-schema');
 var jsf = require('json-schema-faker');
 
-var schemas = require('./schemas');
+var originalSchemas = require('./schemas')
+
+function deepCopy (json) { return JSON.parse(JSON.stringify(json)) }
+// this is an easy way to get immutable schemas and eschew
+// zaggino/z-schema#160 . I think that performance is not a problem
+// for us here
+function getSchemas () { return deepCopy(originalSchemas) }
+var schemas = getSchemas()
```

Comment:
Fine, I suggest a name like `copy` for that function, so we have
var schemas = copy(originalSchemas)
and is more explicit what is going on. 

Up to your, for the rest LGTM , merge :+1: